### PR TITLE
Closes #2140 - Fix description of -InputObject parameter for Measure-…

### LINKED ISSUE #2140: Fix the description of the -InputObject parameter for Measure-Command
Currently, `-InputObject` is described as follows:

> Specifies objects representing the expressions to be measured.

The **actual behavior** is as follows:

Objects bound to `-InputObject` are _optional input_ for the script block passed to `-Expression`:

Inside the script block, `$_` can be used to reference the input object [at hand], as you would in a script block passed to `ForEach-Object`.

Note: If you use the _pipeline_ to pass multiple input objects, `Measure-Command` invokes the `-ScriptBlock` script block _once for each input object_.

While `Measure-Command` still times the _overall_ execution (across multiple script-block invocations, if applicable), the overhead from multiple invocations presumably distorts the timings.

**Therefore, if feasible, pass the input objects as  an (array) _argument_**; e.g.:

```powershell
> Measure-Command { "$($_.Count): [$_]" | Out-Default } -InputObject 'hi', 'there'
2: [hi there]
...
```

That is, the script block is invoked _once_, and  `$_` refers to _all_ input objects.

Contrast this with pipeline use:
```powershell
> 'hi', 'there' | Measure-Command { "$($_.Count): [$_]" | Out-Default }
1: [hi]
1: [there]
...
```

That is, the script block is invoked once _for each input object_, and `$_` refers to the object at hand.

Note that this use of `-InputObject` is a _reversal_ of what makes sense with most other cmdlets, where passing multiple objects only makes sense via the _pipeline_ - see https://github.com/PowerShell/PowerShell/issues/4242.

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### PR DESCRIPTION
**Pull request by bobbytreed**
…Command

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [X] Impacts 6.0 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/3.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/4.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/5.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/6/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806191446144338-2600%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pwh00i2FKxPqqSm1IJV8sHpNfX94MVuPrHCGFvr2AL4%3D&st=2018-06-19T14%3A51%3A19Z&se=2018-07-20T14%3A56%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/3.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/4.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/5.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/6/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806191605265917-2600%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=owxMs6U7LWZAN928e6IB0MAMVota9K4tdhQOuFltHeY%3D&st=2018-06-19T16%3A10%3A45Z&se=2018-07-20T16%3A15%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Utility/Measure-Command.md [hunk 1]
```diff
@@ -112,26 +168,32 @@ Accept wildcard characters: False
...
+
 You can pipe an object to `Measure-Command`.
 
 ## Outputs
 
 ### System.TimeSpan
+
 `Measure-Command` returns a time span object that represents the result.
 
 ## Notes
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Utility/Measure-Command.md]: Remove these notes

Comment (bobbytreed) [reply reference/6/Microsoft.PowerShell.Utility/Measure-Command.md]: Fixed

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/3.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/4.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/5.0/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Measure-Command.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2140/reference/6/Microsoft.PowerShell.Utility/Measure-Command.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806191651510533-2600%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DV9rzsZHxVvGfm6kOkyie8wWNA7dBbB4W3Hr%2FDzwb7Y%3D&st=2018-06-19T16%3A56%3A37Z&se=2018-07-20T17%3A01%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**